app/models/Fairhold.ts

```diff
@@ -7,10 +7,25 @@ const PLATEAU = 0.15
 
 type ConstructorParams = Pick<Fairhold, "affordability" | "landPriceOrRent">;
 
+/** The Fairhold class is a generic class 
+ * (meaning it will calculate a discount regardless if it is given a purchase price or monthly rent).
```

Comment:
```suggestion
/** 
 * The Fairhold class is a generic class 
 * (meaning it will calculate a discount regardless if it is given a purchase price or monthly rent).
```

nit: JSDocs in blocks should have just `/**` as the first line. It's not broken and it does work, but as it's breaking conventions it feels a bit harder to read / unexpected. This applies multiple times throughout the PR below also.

`/** This is the right notation for a single line though 👍  */`

Docs: https://jsdoc.app/about-getting-started

Reply:
Thanks for letting me know, super useful for future reference! Have now fixed this, I believe :)